### ISSUE TITLE
False positive for SC2034 (unused variable)

### ISSUE THREAD

**Issue by brother** [koalaman/shellcheck#138]
I found this warning today, the foo is used to add the default option to the read command in the example, it's not a variable name.

``` shell
read -i "foo" -e -p "Input: " newvalue
        ^-- SC2034: foo appears unused. Verify it or export it.
```

From the help text:
      -e        use Readline to obtain the line in an interactive shell
      -i text   Use TEXT as the initial text for Readline
      -p prompt output the string PROMPT without a trailing newline before
            attempting to read

**Comment by csdfgraham** [comment]
I've found that ShellCheck doesn't report every single occurrence of SC2034 in a file.  You have to fix then one at a time and rescan each time to find them all. Using v4.3.
